CHANGELOG.md

```diff
@@ -5,12 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project
 adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+
+- Adds `click`, `update`, `init` custom events
```

Comment:
How do we imagine them using the `update` and `init` events?

Reply:
Similar to the plan builder events, I was thinking of 2 main scenarios:

First, I can see mainly providers wanting to use these events to gather their own analytics (user viewing/interacting).

Secondly, maybe less popular, but providers might use events to display context-sensitive info based on user selection (e.g. a card appears when something is selected). Or maybe there are warnings they need to display that we don’t know about to be shown before a user signs up (e.g. maybe 2 features semi-conflict with one another and a user has to be educated about something before signing up).

I was also thinking of a third, imaginary, scenario where maybe a provider needs to know about a user’s selection before clicking the CTA. But I’m not really worried about that right now.

I think the reasoning mostly is that we have an event for click, so it should follow we should have similar other events, too, for other key interactions.